docs/rules/no-abusive-eslint-disable.md

```diff
@@ -0,0 +1,43 @@
+# Specify rules to disable when using `eslint-disable` comments
+
```

Comment:
I think we should start with exactly what this does, instead of requiring the users to read until the end to fully understand what this does.